client/web/compose/src/mixins/tabs.js

```diff
@@ -1,12 +1,32 @@
 import { NoID } from '@cortezaproject/corteza-js'
 export default {
+
   methods: {
+    /**
+     * If block has no ID, it is a new block and we need to use tempID
+     * to find it in the list of blocks.
+    */
     fetchID (block) {
       return block.blockID === NoID ? block.meta.tempID : block.blockID
     },
 
+    /**
+     * Find block by ID or tempID
+     * You can find a block by passing a block into fetchID
+     * before passing it into findBlock
+     */
     findBlock (id) {
```

Comment:
Why doesn't findBlock just check both values? if it matches either its returned no?